src/main/java/Engineer/Engineer.java

```diff
@@ -0,0 +1,93 @@
+package engineer;
+import java.util.Scanner;
+import engineer.exception.EngineerException;
+import engineer.task.*;
+import engineer.command.*;
+
+public class Engineer {
+
+    static Scanner scanner = new Scanner(System.in);
+    static TaskManager myTaskManager = new TaskManager();
+    static ChatBotManager myChatBotManager = new ChatBotManager();
+    static TextDivider myTextDivider = new TextDivider();
+
+    public static void printResponse() throws EngineerException{
```

Comment:
Few things about this method
1. The method is quite long (>30LOC). Consider chunking it into multiple methods to maintain redability
2. The naming is not apt because it is not only printing the response but doing several other tasks within it.